test/lib/index.test.js

```diff
@@ -27,6 +28,7 @@ describe('ImageAsset', function() {
             "matchingLinks": [ 'https://www.example.com/' ],
 
             "title": 'Test Asset',
+            "synopsis": 'Test Asset synopsis',
```

Comment:
@lmyllari You don't need mocking for the basics:
For each asset type:
 - New one up, set thumbnail to fake val, and ensure that value returns in get_metadata call
 - Same with synopsis
 - For asset types with extra dependent ids, set a fake doc and verify that get_dependent_assets returns those along with thumbnail